content/desktop/release-notes.md

```diff
@@ -24,11 +24,74 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+# 4.24.0
+
+{{< release-date date="2023-09-28" >}}
+
+{{< desktop-install all=true version="4.24.0" build_path="/" >}}
+
+### New
+
+- New Notification Center available to all users.
+- Resource Saver available to all users. This feature can be configured from the **Resources** tab in **Settings**. For more information see [Docker Desktop's Resource Saver mode](resource-saver.md)
+
+### Upgrades
+
+- [Compose v2.22.0](https://github.com/docker/compose/releases/tag/v2.22.0)
+- [Go 1.21.1](https://github.com/golang/go/releases/tag/go1.21.1)
+- [Wasm](../desktop/wasm/_index.md) runtimes:
+  - wasmtime, wasmedge `v0.2.0`.
+  - lunatic, slight, spin, and wws`v0.9.1`.
+  - Added wasmer wasm shims.
+
+
+### Bug fixes and enhancements
+
+#### For all platforms
+
+- Docker Init:
+  - Fixed an issue formatting Dockerfile file paths for ASP.NET projects on Windows.
+  - Improved performance on language detection for large directories with lots of files.
+- Added a timeout to polling for resource usage stats used by the Containers dashboard. Fixes [docker/for-mac#6962](https://github.com/docker/for-mac/issues/6962).
+- containerd integration:
+  - Implemented push/pull/save image events.
+  - Implemented pulling legacy schema1 images.
+  - Implemented `docker push --all-tags`.
+  - Implemented counting containers using a specific image (visible for example in `docker system df -v`).
+  - Validated pulled image names are not reserved.
+  - Handle `userns-remap` daemon setting.
+  - Fixed legacy builder build errors when multiple COPY/ADD instructions are used.
+  - Fixed `docker load` causing pool corruption which could some subsequent image related operations.
+  - Fixed not being able to reference images via truncated digest with a `sha256:` prefix.
+  - Fixed `docker images` (without `--all`) showing intermediate layers (created by the legacy classic builder).
+  - Fixed `docker diff` containing extra differences.
+  - Changed `docker pull` output to match the output with containerd integration disabled.
+- You can now view and manage the Docker Engine state from the dashboard.
```

Comment:
```suggestion
- You can now view and manage the Docker Engine state, with pause, stop, and resume, directly from the Docker Dashboard.
```

Reply:
Think this should also be moved to the 'new' section since this is featured quite strongly in the release blog